include/nil/actor/testing/thread_test_case.hh

```diff
@@ -49,3 +49,25 @@
     void name::do_run_test_case()
 
 #define ACTOR_THREAD_TEST_CASE(name) ACTOR_THREAD_TEST_CASE_EXPECTED_FAILURES(name, 0)
+
+#define ACTOR_FIXTURE_TEST_CASE_EXPECTED_FAILURES(name, fixture, failures)      \
+    struct name : public nil::actor::testing::actor_test, public fixture {            \
```

Comment:
I think - maybe we can remove so many code duplicate? 

Difference between fixture/thread test case only in inheritance from fixture.
So let's just make one define 
```
#define ACTOR_FIXTURE_TEST_CASE_EXPECTED_FAILURES(name, fixture, failures) 
   struct name : public nil::actor::testing::actor_test, public fixture { ... } 
```
and after this define all what we need like 
```
#define ACTOR_FIXTURE_TEST_CASE(name, fixture) ACTOR_FIXTURE_TEST_CASE_EXPECTED_FAILURES(name, fixture, 0)
#define ACTOR_THREAD_TEST_CASE(name) ACTOR_FIXTURE_TEST_CASE_EXPECTED_FAILURES(name, dump_fixture, 0)
#define ACTOR_THREAD_TEST_CASE_EXPECTED_FAILURES(name, failures) ACTOR_FIXTURE_TEST_CASE_EXPECTED_FAILURES(name, dump_fixture, failures)
```
where ```dump_fixture: struct dump_fixture {} ```

Reply:
Changed.